cmux-tui/scripts/smoke-windows-conpty-resize.py

```
@@ -0,0 +1,197 @@
+        time.sleep(0.01)
+    raise TimeoutError(f"server did not publish {socket_path}")
+
+
+def wait_for_first_output(pty: PtyProcess) -> None:
+    events: queue.Queue[BaseException | None] = queue.Queue()
+
+    def read() -> None:
+        try:
+            chunk = pty.read(8192)
+            if not chunk:
+                raise RuntimeError("attach closed before it produced output")
```

Ruff (0.16.1)

[warning] Abstract `raise` to an inner function

(TRY301)